### ISSUE TITLE
New relationship properties and classes

### ISSUE THREAD

**Issue by jodiw01** [lcnetdev/bibframe-ontology#116]
https://github.com/lcnetdev/bflc-ontology/issues/22

Transition relationship properties and classes from bflc to BF

Labels and definitions may be changed as well.

**Comment by jodiw01** [comment]
**Add relation, Relation, relationship, and Relationship to the BF vocabulary**

This Github issue introduces a way to express relationships between resources indirectly. The labels and definitions that were originally modelled in BFLC have been modified.

In 2017, The Library of Congress began experimenting with abstracting relationships when an indirect method was introduced to the BFLC extension/test ontology.  The indirect method introduced a pattern akin to the bf:contribution/bf:Contribution construct.

Contribution:
```
<bf:Work rdf:about="http://id.loc.gov/resources/works/5625322">
	<bf:contribution>
		<bf:Contribution>
			<bf:agent rdf:resource="http://id.loc.gov/rwo/agents/n81118760"/>
			<bf:role rdf:resource="http://id.loc.gov/vocabulary/relators/cmp"/>
		</bf:Contribution>
    </bf:contribution>
</bf:Work>
```

Indirect relationship:
```
<bf:Work rdf:about="http://id.loc.gov/resources/works/5625322">
	<bf:relation>
		<bf:Relation>
			<bf:relationship rdf:resource="http://id.loc.gov/entities/relationships/contains">
			<bf:relatedResource rdf:about="http://id.loc.gov/resources/hubs/c580e172-4831-149b-d96f-9f711ed9bfe8"
		<bf:Relation>
    </bf:relation>
</bf:Work>
```

The experimental indirect method was developed to: 

  * Better accommodate RDA cataloging, where there was no guarantee of a URI for a given relationship
  * More readily handle thousands of RDA relationships
  * Handle variance in converted MARC
  * Accommodate extra relationship information.

It was impractical to use the direct method in most of these cases and impossible for one or two cases.  
The indirect method has addressed those challenges and exceeded them in many cases, meaning it was found to be useful beyond the initial use cases.  These findings were bolstered by feedback from other BIBFRAME implementers and groups, such as National Library of Sweden, OCLC, and PCC BIG, to name three. 

One point of discussion was around reification.  While this additional abstraction is a form of RDF reification, the parity here with the BIBFRAME Contribution model argues against introducing a different form at this time.  Also, any formal reification suggestions that may accompany RDF 1.2 are still being discussed.

Expected/recommended changes:

* **bf:relation** – new property (formerly bflc:relationship)
	Label: 		relation
	Definition: 	Related resource and its association with another resource.
	Domain:  	Suggested use - With Hub, Work, Instance, Item or Event
	Range: 	 	bf:Relation
	SubPropertyOf:	None
	
* **bf:Relation** – new class (formerly bflc:Relationship)
	Label: 		Relation
	Definition:  	Related resource and its association with another resource.
	SubClassOf: 	None

* **bf:relationship** – new property (formerly bflc:relation)
	Label: 		relationship
	Definition: 	Type of association between resources.
	Domain:  	bf:Relation
	Range:  		bf:Relationship
	SubPropertyOf: 	None
	Editorial Note: 	Any entry from the Relationships vocabulary [http://id.loc.gov/vocabulary/relationship] may be used; all have been defined as a bf:Relationship.  This vocabulary is not inclusive; others may be employed.

* **bf:Relationship** – new class (formerly bflc:Relation)
	Label: 		Relationship
	Definition: 	Type of association between resources.
	SubClassOf: 	None
	
* **bf:relatedResource** – new property
	Label: 		Related resource
	Definition: 	Related resource
	Domain:  	bf:Relation
	Range:  		rdf:Resource

Expected use

The Library of Congress plans to adopt these changes and will update its conversion programs and systems accordingly.

**Comment by frosterus** [comment]
Thank you for this change - seems excellent!

Three questions:

1. Will the newly introduced method replace the old one? In other words, will the subproperties of bf:relatedTo be deprecated?
2. What is the difference between [bf:relatedTo](https://id.loc.gov/ontologies/bibframe.html#p_relatedTo) and [bf:associatedResource](https://id.loc.gov/ontologies/bibframe.html#p_associatedResource)? The definition for the latter would be useful.
3. Should the Suggeted value for bf:associatedResource include Instance (and maybe Item as well)? For example, [bf:hasReproduction](https://id.loc.gov/ontologies/bibframe.html#p_hasReproduction) (rdam:P30039 in RDA) is between Instances.

**Comment by jodiw01** [comment]
included in v2.4
